### PR TITLE
Added CompleteCaseForm, cleanup some code

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/__mocked__/apiSchema.d.ts [hunk 1]
```diff
@@ -3,9 +3,15 @@ declare namespace MockComponents {
 
     export type Case = {
       id: number
+      teamTitle: string
```
Comment (gilleswittenberg) [inline src/__mocked__/apiSchema.d.ts:6]: Think `team: { id: number, title: string }` will be more close to eventual API data structure

File: src/app/features/cases/components/CaseDetails/CaseDetails.tsx [hunk 2]
```diff
@@ -4,15 +4,15 @@ import { useCase } from "app/state/rest"
 import { displayDate } from "app/features/shared/components/atoms/DateDisplay/DateDisplay"
 
 type Props = {
-  caseId: Components.Schemas.CaseEvent["id"]
+  caseId: MockComponents.Schemas.Case["id"]
```
Comment (gilleswittenberg) [inline src/app/features/cases/components/CaseDetails/CaseDetails.tsx:7]: `Components.Schemas.Case["id"]` is the same type

File: src/app/pages/cases/details/DetailsPage.tsx [hunk 3]
```diff
@@ -59,6 +59,10 @@ const DetailsPage: React.FC<RouteComponentProps<Props>> = ({ id }) => {
             <ButtonLink to={ to("/cases/:id/correspondence", { id: id })}>
               <ButtonTertiary variant="tertiary">Correspondentie</ButtonTertiary>
             </ButtonLink>
+            &nbsp;
```
Comment (gilleswittenberg) [inline src/app/pages/cases/details/DetailsPage.tsx:62]: Can we solve this with CSS (horizontal padding / margin)?

**Review by gilleswittenberg — APPROVED** [review]

File: src/__mocked__/apiSchema.d.ts (hunk 1 above)
Comment (remcohoff) [reply src/__mocked__/apiSchema.d.ts:6]: True, fixed it

File: src/app/features/cases/components/CaseDetails/CaseDetails.tsx (hunk 2 above)
Comment (remcohoff) [reply src/app/features/cases/components/CaseDetails/CaseDetails.tsx:7]: Fixed it

File: src/app/pages/cases/details/DetailsPage.tsx (hunk 3 above)
Comment (remcohoff) [reply src/app/pages/cases/details/DetailsPage.tsx:62]: Yes, fixed it

**Pull request merged**